src/magnify/apps/core/models.py

```diff
@@ -15,3 +17,98 @@ class Meta:
 
     def __str__(self):
         return self.username
+
+class Meeting(models.Model):
+    """Model for one meeting or a collection of meetings defined reccursively"""
+    name = models.CharField(max_length=500)
+
+    # Start and end are the same for a single meeting
+    start = models.DateTimeField()
+    end = models.DateTimeField()
```

Comment:
Now that the time is a separate field, these should be of type `DateField`... I know I asked the opposite but that's when I didn't understand the model usage yet :hand_over_mouth: